src/style.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.wrapper {
+  position: relative;
+  display: block;
+  margin-top: 20px;
+  min-width: 100px;
+}
+
+.icon {
+  position: absolute;
+  top: 25px;
+  left: 26px;
+}
+
+.search {
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300px;
+  font-size: 16px;
+  line-height: 22px;
+  box-sizing: border-box;
+  height: 70px;
+  padding-left: 62px;
+}
+
+.search:hover {
+  color: #3d4e61;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-family: "Avenir-Heavy", sans-serif;
+  color: #3d4e61;
```

Comment:
add `color` property to default class `.search` so that you dont have to add on `.search:focus ` and `.search:hover `